test/Thermal/piston.jl

```diff
@@ -1,4 +1,4 @@
-using ModelingToolkit, OrdinaryDiffEqDefault, Test
+using ModelingToolkit, Test
```

Comment:
what's the point of the removal if OrdinaryDiffEq is directly used in others?

Reply:
I use TestEnv.jl yoninstantiate the test environment, which is generally different from the package environment, notably, tests often require additional dependencies.